src/Avalonia.Base/Media/Fonts/CustomFontCollection.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using Avalonia.Platform;
+
+namespace Avalonia.Media.Fonts
+{
+    public class CustomFontCollection : FontCollectionBase
+    {
+        protected List<FontFamily> _fontFamilies { get; } = new List<FontFamily>();
```

Comment:
Is this meant to be protected for public manipulation? If so, perhaps it shouldn't start with `_`